src/main/java/org/broadinstitute/consent/http/resources/DataAccessRequestResource.java

```diff
@@ -575,6 +570,55 @@ private DataAccessRequest populateDarFromJsonString(User user, String json) {
     return newDar;
   }
 
+  /**
+   * Populate a new Data Access Request from the JSON string and the parent Data Access Request.
+   * Copies all the data from the parent dar, then overwrites the collaborators and datasets.
+   * Adds all progress report specific fields.
+   *
+   * @param json      The JSON string to populate the new Progress Report.
+   * @param parentDar The parent Data Access Request to copy data from.
+   * @return A new Progress Report populated with the provided JSON string and parent DAR data.
+   */
+  public DataAccessRequest populateProgressReportFromJsonString(String json, DataAccessRequest parentDar) {
```

Comment:
This feels more like a service layer method than a resource layer method, but it is building off of shared code in `populateDARData` (👍🏽) so I think it's fine here without doing a larger refactor.